lib/typescript/render/providers/facebook/components/InstagramStoryMention/index.module.scss

```diff
@@ -0,0 +1,43 @@
+@import 'assets/scss/fonts.scss';
+@import 'assets/scss/colors.scss';
+
+.textMessage {
+  display: inline-block;
+  overflow-wrap: break-word;
+  max-width: 500px;
+  word-break: break-word;
+  white-space: pre-wrap;
+  padding: 10px;
+  margin-top: 5px;
+  border-radius: 8px;
+  font-family: 'Lato', sans-serif;
+}
+
+.activeStory {
+  display: flex;
+  align-items: center;
```

Comment:
Instead of `L21`, sass also let's you put:
```sass
&:link, &:hover, &:visited, &:active {
  color: var(--color-text-contrast);
  font-style: italic;
}
```
(matter of taste though no need to change)